src/components/FloatingDropdown/FloatingDropdown.tsx

```diff
@@ -156,7 +171,7 @@ export class FloatingDropdown extends React.Component<FloatingDropdownProps> {
         disabled={disabled}
         options={coreOptions}
         placement={placement}
-        onExpandedChange={onExpandedChange}
+        onExpandedChange={this.onExpandedChange}
```

Comment:
Can you please remove the `onExpandedChange` from the const at the top of the function?
(line 134)